samples/csharp/getting-started/MulticlassClassification_Iris/IrisClassification/IrisClassificationConsoleApp/Program.cs

```diff
@@ -48,10 +48,10 @@ private static void BuildTrainEvaluateAndSaveModel(MLContext mlContext)
             
 
             // STEP 2: Common data process configuration with pipeline data transformations
-            var dataProcessPipeline = mlContext.Transforms.Concatenate(DefaultColumnNames.Features, "SepalLength",
-                                                                                   "SepalWidth",
-                                                                                   "PetalLength",
-                                                                                   "PetalWidth")
+            var dataProcessPipeline = mlContext.Transforms.Concatenate(DefaultColumnNames.Features, nameof(IrisData.SepalLength),
```

Comment:
Refactored so that we can avoid runtime errors by replccing strings with strong types. I missed it in v0.10 migraion.